src/app/hooks/use-metadata-refresh.ts

```diff
@@ -38,8 +38,10 @@ export const useMetadataRefresh = () => {
       return;
     }
 
-    if (needToSetVersion)
-      fetchTokensMetadata(tezos.rpc.getRpcUrl(), slugsOnAppLoad)
+    if (!needToSetVersion) return;
+
+    if (chainId === TempleChainId.Mainnet) {
+      fetchTokensMetadata(chainId, slugsOnAppLoad)
```

Comment:
Please check how tokens from Ghostnet are loaded. If everything is fine both with an old account and a recently imported account, just resolve this conversation

Reply:
it's fine